sdk/servicebus/service-bus/package.json

```diff
@@ -46,6 +46,8 @@
     "@types/mocha": "^5.2.5",
     "@types/node": "^8.0.37",
     "@types/ws": "^6.0.1",
+    "@typescript-eslint/eslint-plugin": "^1.7.0",
+    "@typescript-eslint/parser": "^1.7.0",
```

Comment:
It looks like a lot of eslint dependencies are missing from here.

Reply:
I was having a rebase nightmare yesterday. This might be a result of that

Reply:
Fixed the missing dependencies